tests/test_chiral_restraints.py

```diff
@@ -0,0 +1,526 @@
+import multiprocessing
+import os
+
+os.environ["XLA_FLAGS"] = "--xla_force_host_platform_device_count=" + str(multiprocessing.cpu_count())
```

Comment:
last time I checked, `jax.vmap` doesn't efficiently push through `jax.while_loop` (it will still run, but you won't get any speed-up, and the cpu usage is pegged to 1-core)

Reply:
Sorry didn't see "on the test-runner", I have no idea, but locally (on my mac and desktop), its a 10x speed-up.